src/common/plugins/plugins-config.js

```diff
@@ -141,6 +143,30 @@ function getEncodedReferrer(): string {
   return encodeURIComponent(referrer);
 }
 
+/**
+ * @private
+ * @param {string} text - the string to sanitize
+ * @returns {string} - the sanitized string
+ * @private
+ */
+function _sanitize(text: string): string {
+  if (!text) return;
```

Comment:
@lianbenjamin  this line fail in flow 
https://app.travis-ci.com/github/kaltura/kaltura-player-js/jobs/583634774